File: python/count_characters.py
def count_characters(file_path):
    with open(file_path, 'r', encoding='utf-8') as f:
        content = f.read()
        return len(content)

# Exemple d'utilisation
filename = "../files/count_characters.txt"
print("Nombre de caractères :", count_characters(filename))

File: c/count_characters.c
#include <stdio.h>

int main() {
    FILE *file = fopen("../files/count_characters.txt", "r");
    if (!file) {
        perror("Erreur d'ouverture du fichier");
        return 1;
    }

    int count = 0;
    while (fgetc(file) != EOF) {
        count++;
    }

    fclose(file);
    printf("Nombre de caractères : %d\n", count);
    return 0;
}
